dandiapi/api/views/audit.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import serializers
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.pagination import CursorPagination
+
+from dandiapi.api.models import AuditRecord
+from dandiapi.api.permissions import IsApproved
+
+if TYPE_CHECKING:
+    from rest_framework.request import Request
+    from rest_framework.response import Response
+
+
+class AssetAuditEventQuerySerializer(serializers.Serializer):
+    before = serializers.DateTimeField(allow_null=True, default=None)
+    after = serializers.DateTimeField(allow_null=True, default=None)
+    reverse_order = serializers.BooleanField(default=False)
+
+
+class AssetAuditEventResponseSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = AuditRecord
+        fields = [
+            'id',
+            'dandiset_id',
+            'timestamp',
+            'record_type',
+            'details',
+        ]
+
+    details = serializers.SerializerMethodField()
+
+    # Don't need to return metadata
+    def get_details(self, obj: AuditRecord):
+        obj.details.pop('metadata', None)
+        return obj.details
+
+
+class AssetuditEventPagination(CursorPagination):
+    ordering = 'timestamp'
+    page_size = 100
+    page_size_query_param = 'page_size'
+    max_page_size = 1000
+
+
+@swagger_auto_schema(
+    methods=['GET'],
+    responses={200: 'The asset audit events'},
+)
+@api_view(['GET'])
+@permission_classes([IsApproved])
+def asset_audit_events(request: Request) -> Response:
+    queryset = AuditRecord.objects.filter(record_type__in=['add_asset', 'update_asset'])
```

Comment:
Actually, could I request you add `delete_asset` (or w/e the name of the event is) as well?